x/auth/module.go

```diff
@@ -68,17 +68,14 @@ func (AppModuleBasic) GetQueryCmd(cdc *codec.Codec) *cobra.Command {
 // app module object
 type AppModule struct {
```

Comment:
The auth module, like other modules that impact supply, should also accept a `SupplyKeeper` (based on an expected interface -- not directly) instead of doing this ad-hoc method of a pseudo-module account.